pkg/apis/certificates/validation/validation.go

```diff
@@ -203,7 +165,7 @@ func validateCertificateSigningRequest(csr *certificates.CertificateSigningReque
 	allErrs = append(allErrs, validateConditions(field.NewPath("status", "conditions"), csr, opts)...)
 
 	if !opts.allowArbitraryCertificate {
-		if err := validateCertificate(csr.Status.Certificate); err != nil {
+		if err := utilcert.ValidateCertificatePEM(csr.Status.Certificate); err != nil {
```

Comment:
I don't think we should move this validation out of the APIs package to a shared function that can accidentally be modified in ways that break the API